containers/roles/certificates/tasks/ca.yml

```diff
@@ -1,8 +1,3 @@
-- name: 'Install openssl'
-  yum:
```

Comment:
Is this no longer needed or are you now assuming it's present? On the latter it might be better to use ```package``` instead.

Reply:
This annoyingly causes breakages on Travis due to it having to use the apt module. I can add it back and you can see the error. My googling indicated no one had solved it.